### PR TITLE
docs: add user guides for use grpc basic auth

### PR DESCRIPTION
**Pull request by Ja7ad**

### CODE REVIEW

**Comment by Ja7ad** [comment]
@Mr-HoDL58 @b00f I fixed errors, please review this post.

File: website/_i18n/zh/user-guides/grpc-basic-auth/index.md [hunk 1]
```diff
@@ -23,6 +23,32 @@ password: bar
...
+- `-n`: Output the hashed password to the console rather than updating a file.
+- `-B`: Force the use of the bcrypt encryption algorithm.
+- `-C cost`: Set the cost factor for the bcrypt algorithm. Higher values result in slower hashing but are more secure.
+- `username`: The username for which you are generating the password.
+- `password`: The password you wish to hash.
+
+Example:
+
+```shell
+htpasswd -bnBC 10 javad mypassword
```
Comment (b00f) [inline website/_i18n/zh/user-guides/grpc-basic-auth/index.md:48]:
```suggestion
htpasswd -bnBC 10 user pass
```

File: website/_i18n/en/user-guides/grpc-basic-auth/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+- `-n`: Output the hashed password to the console rather than updating a file.
+- `-B`: Force the use of the bcrypt encryption algorithm.
+- `-C cost`: Set the cost factor for the bcrypt algorithm. Higher values result in slower hashing but are more secure.
+- `username`: The username for which you are generating the password.
+- `password`: The password you wish to hash.
+
+Example:
+
+```shell
+htpasswd -bnBC 10 javad mypassword
```
Comment (b00f) [inline website/_i18n/en/user-guides/grpc-basic-auth/index.md]:
```suggestion
htpasswd -bnBC 10 user pass
```

File: website/_i18n/zh/user-guides/grpc-basic-auth/index.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
```
Comment (b00f) [inline website/_i18n/zh/user-guides/grpc-basic-auth/index.md]:
```suggestion
htpasswd -bnBC 10 user pass
```

File: website/_i18n/ar/user-guides/grpc-basic-auth/index.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+- `-n`: Output the hashed password to the console rather than updating a file.
+- `-B`: Force the use of the bcrypt encryption algorithm.
+- `-C cost`: Set the cost factor for the bcrypt algorithm. Higher values result in slower hashing but are more secure.
+- `username`: The username for which you are generating the password.
+- `password`: The password you wish to hash.
+
+Example:
+
+```shell
+htpasswd -bnBC 10 javad mypassword
```
Comment (b00f) [inline website/_i18n/ar/user-guides/grpc-basic-auth/index.md]:
```suggestion
htpasswd -bnBC 10 user pass
```

**Review by b00f — CHANGES_REQUESTED** [review]

**Comment by Ja7ad** [comment]
@b00f I fix all things

**Review by b00f — APPROVED** [review]

**Pull request merged**